apps/settings/js/panels/operator_settings/views/operator_item_template.js

```diff
@@ -0,0 +1,51 @@
+define(function() {
```

Comment:
Let's put `require` here to make sure we won't break anything in the future even if we don't require anything now.